### PR TITLE
monitor: add platform metrics

### PR DESCRIPTION
**Pull request by huanggze**
Signed-off-by: huanggze <loganhuang@yunify.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

 /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

- Add platform metrics:
```
GET /kapis/monitoring.kubesphere.io/v1alpha3/kubesphere
---
kubesphere_workspace_count
kubesphere_user_count
kubesphere_cluser_count
kubesphere_app_template_count
```
- Add back workspace statitics data:
```
GET /kapis/monitoring.kubesphere.io/v1alpha3/workspace/{workspace}?type=statistics
---
workspace_namespace_count
workspace_devops_project_count
workspace_member_count
workspace_role_count
```

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/kapis/monitoring/v1alpha3/handler.go [hunk 1]
```diff
@@ -33,8 +35,13 @@ type handler struct {
 	mo model.MonitoringOperator
 }
 
-func newHandler(k kubernetes.Interface, m monitoring.Interface) *handler {
-	return &handler{k, model.NewMonitoringOperator(m)}
+func newHandler(k kubernetes.Interface, m monitoring.Interface, f informers.InformerFactory, o openpitrix.Client) *handler {
+	return &handler{k, model.NewMonitoringOperator(m, k, f, o)}
+}
+
+func (h handler) handlePlatformMetricsQuery(req *restful.Request, resp *restful.Response) {
```
Comment (benjaminhuo) [inline pkg/kapis/monitoring/v1alpha3/handler.go:42]: handlePlatformMetricsQuery  => handleKubeSphereMetricsQuery?

Comment (huanggze) [reply pkg/kapis/monitoring/v1alpha3/handler.go]: done

File: pkg/kapis/monitoring/v1alpha3/register.go [hunk 2]
```diff
@@ -36,10 +38,18 @@ const (
 
 var GroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1alpha3"}
 
-func AddToContainer(c *restful.Container, k8sClient kubernetes.Interface, monitoringClient monitoring.Interface) error {
+func AddToContainer(c *restful.Container, k8sClient kubernetes.Interface, monitoringClient monitoring.Interface, factory informers.InformerFactory, opClient openpitrix.Client) error {
 	ws := runtime.NewWebService(GroupVersion)
 
-	h := newHandler(k8sClient, monitoringClient)
+	h := newHandler(k8sClient, monitoringClient, factory, opClient)
```
Comment (zryfish) [inline pkg/kapis/monitoring/v1alpha3/register.go:44]: What will happen if `opClient` is nil

Comment (huanggze) [reply pkg/kapis/monitoring/v1alpha3/register.go:44]: return error 

```
{
  results: [
    {
      metric_name: kubesphere_xxx_count
      data: ...
    },
   {
      metric_name: kubesphere_app_template_count
      error: ...
    }
  ]
}
```

**Pull request merged**